spec/features/user_session_timeout_spec.rb

```diff
@@ -0,0 +1,40 @@
+require 'features/support/sign_up_helpers'
+require 'features/support/not_signed_in'
+require 'timecop'
+
+describe 'with devise timeoutable enabled' do
+  after { Timecop.return }
+
+  context 'when user has not signed in' do
```

Comment:
Is this covered by other tests well enough? Don't think we're providing a unique context here.